README.md

```diff
@@ -18,6 +18,62 @@ dev_appserver .
 
 See [CONTRIBUTING.md](/CONTRIBUTING.md) for more information on local development.
 
+## Using the data
+
+All test result data is public. There are two types of gzipped JSON data files we store.
+
+### Test run summary files
+
+These are of the pattern: `{sha[0:10]}/{platform_id}-summary.json.gz`
+
+- `sha[0:10]` - the first 10 characters of the WPT SHA that run was tested against
+- `platform_id` - the key of the platform configuration in `browsers.json`
+
+Example: https://storage.googleapis.com/wptd/791e95323d/firefox-56.0-linux-summary.json.gz
```

Comment:
Examples are always helpful :-)

If it's important and expected to say the same, we could also note that the bucket is `wptd`. (Is that the bucket name?)